packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -203,8 +203,8 @@ export default class LabeledTextField extends React.Component<Props, State> {
                                 value={this.state.value}
                                 placeholder={placeholder}
                                 disabled={disabled}
-                                validation={validation}
-                                onValidation={this.handleValidation}
+                                validation={validate}
+                                onValidation={this.handleValidate}
```

Comment:
I will be updating the props here when I rename them to `validate` and `onValidate` in `TextField` as well (See this [ticket](https://khanacademy.atlassian.net/browse/WB-1071)).